modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -10192,6 +10192,8 @@ oauth2-allow-authorization-code-pkce-grant=Allow PKCE Extended Authorization Cod
 oauth2-allow-authorization-code-pkce-grant-description=When unselected, remote clients cannot request new tokens, applies immediately to existing and new clients.
 oauth2-allow-client-credentials-grant=Allow Client Credentials
 oauth2-allow-client-credentials-grant-description=When unselected, remote clients cannot request new tokens, applies immediately to existing and new clients.
+oauth2-allow-jwt-bearer-grant=Allow JWT Bearer Grant
```

Comment:
Why are these lang keys added back in this commit? I thought you wanted the PR sent without them?

Reply:
That was a mistake, thanks for spotting it! Force pushing...